src/NLog/Internal/Fakeables/AppDomainWrapper.cs

```diff
@@ -102,23 +100,19 @@ public AppDomainWrapper(AppDomain appDomain)
         /// <summary>
         /// Process exit event.
         /// </summary>
-        public event EventHandler<EventArgs> ProcessExit;
+        public event EventHandler ProcessExit
```

Comment:
Don't know if it is breaking change, but AppDomainWrapper resides within NLog/Internals, so would be surprised if anyone used it.

Alternative solution would be to make AppDomainWrapper IDisposable and then ensure to Dispose()  everywhere.